cmd/kube-aggregator/hack/update-codegen.sh

```diff
@@ -67,3 +76,12 @@ ${informergen} \
   --listers-package k8s.io/kubernetes/cmd/kube-aggregator/pkg/client/listers \
   --output-package k8s.io/kubernetes/cmd/kube-aggregator/pkg/client/informers
   "$@"
+
+# this is a temporary hack until we manage to update codegen to accept a scheme instead of hardcoding it
+echo "rewriting imports"
+grep -Rl "\"k8s.io/kubernetes/pkg" "${KUBE_ROOT}/cmd/kube-aggregator/pkg/client" | \
```

Comment:
no `-1` in BSD grep. This works: `grep -R -H <pattern> <dir> | cut -d: -f1 | sort | uniq`